Content.IntegrationTests/Tests/Interaction/InteractionTest.Constants.cs

```diff
@@ -27,8 +28,12 @@ public abstract partial class InteractionTest
 
     // Parts
     protected const string Bin1 = "MatterBinStockPart";
+    protected const string Bin4 = "BluespaceMatterBinStockPart";
     protected const string Cap1 = "CapacitorStockPart";
+    protected const string Cap4 = "QuadraticCapacitorStockPart";
     protected const string Manipulator1 = "MicroManipulatorStockPart";
+    protected const string Manipulator4 = "FemtoManipulatorStockPart";
     protected const string Battery1 = "PowerCellSmall";
     protected const string Battery4 = "PowerCellHyper";
```

Comment:
Should validate these
also why does this exist at all

Reply:
Love that this PR adds a Shit-tier useless test. I'll probably just yeet the test it adds because it's just genuinely worthless.